components/cdc/tests/failpoints/test_observe.rs

```diff
@@ -125,3 +133,85 @@ fn test_stale_observe_cmd() {
     event_feed_wrap.as_ref().replace(None);
     suite.stop();
 }
+
+#[test]
+fn test_delayed_change_cmd() {
```

Comment:
What does it want to test?

Reply:
For testing change cmd which contains a stale observe id.